app/pages/schedule.vue

```diff
@@ -22,3 +122,180 @@ useSeoMeta({
     ogImage: computed(() => prismic.asImageSrc(page.value?.data.meta_image)),
 });
 </script>
+
+<style lang="postcss" scoped>
+.page-container {
+    max-width: 1240px;
+    min-height: 100vh;
+    margin: auto;
+    padding: 0 16px;
+}
+.date-tabs {
+    display: inline-flex;
+    gap: 8px;
+    border: 1px solid var(--gray-900);
+    border-radius: 8px;
+    padding: 8px;
+    margin: 0 0 64px;
+    list-style: none;
+}
+.date-tab {
+    font-size: rem(14px);
+    line-height: 1.25;
+    color: var(--gray-900);
+    background-color: transparent;
+    border-radius: 6px;
+    padding: 8px;
+    line-height: 1.5;
+    cursor: pointer;
+    border: 2px solid transparent;
+    text-wrap: balance;
+    @media (--lg) {
+        font-size: rem(16px);
+        padding: 10px 12px;
+    }
+    transition:
+        background-color var(--hover-transition),
+        color var(--hover-transition),
+        border-style var(--hover-transition),
+        border-color var(--hover-transition);
+    &.is-active {
+        background-color: var(--gray-900);
+        color: var(--beige-100);
+        border-color: var(--gray-900);
+    }
+    &:hover,
+    &:focus-visible {
+        background-color: var(--beige-100);
+        color: var(--gray-900);
+        border-color: var(--gray-900);
+        border-style: dashed;
+    }
+}
+.schedule-grid {
+    margin-bottom: 64px;
+}
+.timeslot {
+    @media (--md) {
+        display: grid;
+        grid-template-columns: 80px 1fr;
+    }
+    @media (--lg) {
+        &.regular {
+            + .regular {
+                .timeslot-sessions {
+                    border-top: 0;
+                }
+            }
+            &:has(+ .special) {
```

Comment:
`&:has(+ .special) {` le + est surement erreur ?
Has ça regarde les enfants, le + et le ~ ça regarde les sibllbings

Reply:
Non ça fonctionne comme voulu :)

Reply:
I still think there is something wrong 🤔 C'est weird en tk